gridstatus/miso.py

```diff
@@ -1323,35 +1395,86 @@ def _get_constraint_header_dates_from_excel(
         )
         return market_date, publish_date
 
+    @support_date_range(frequency="DAY_START")
+    def get_look_ahead_hourly(
+        self,
+        date: str | pd.Timestamp,
+        end: str | pd.Timestamp | None = None,
+        verbose: bool = False,
+    ) -> pd.DataFrame:
+        if date == "latest":
+            return self.get_look_ahead_hourly(date="today", verbose=verbose)
 
-def add_interval_end(df: pd.DataFrame, duration_min: int) -> pd.DataFrame:
-    """Add an interval end column to a dataframe
+        url = f"https://docs.misoenergy.org/marketreports/{date.strftime('%Y%m%d')}_sr_la_rg.csv"
+        logger.info(f"Downloading look-ahead hourly data from {url}")
+        publish_date = pd.read_csv(
+            url,
+            nrows=1,
+            skiprows=1,
+            header=None,
+        ).iloc[0, 0]
+        publish_time = pd.to_datetime(publish_date.split(": ")[1]).tz_localize(
+            self.default_timezone,
+        )
 
-    Args:
-        df (pandas.DataFrame): Dataframe with a time column
-        duration_min (int): Interval duration in minutes
+        df_raw = pd.read_csv(url, skiprows=3)
+        df = df_raw.iloc[:96]  # 24 hours * 4 regions per hour
```

Comment:
Nit: if the number of rows to ignore at the bottom of the csv are the same every time, we could use `skipfooter` in the `read_csv` call.